### ISSUE TITLE
New Settings page

### ISSUE THREAD

**Issue by icrc-fdeniger** [icrc/openmrs-android-fhir#67]
A new Settings page should be introduced to allow users to modify their preferences. This page will offer the following features:

 **Check Connectivity at Startup:**

Option for users to enable or disable the automatic check for internet connectivity when the app starts.

**Server Connectivity & Data Sync Notifications:**

Option to enable or disable notifications about server connectivity and data synchronization.
If enabled, users can set the frequency of these notifications (e.g., daily, weekly, or custom intervals).

**Developer Documentation for Adding New Options**
To enable developers to easily extend the settings page with additional options, the following steps should be documented:

1. Define a New Preference:
Add new keys for preferences in the app's SharedPreferences or DataStore (e.g., is_notifications_enabled, sync_frequency).
Each key should have a default value defined in the app's configuration.
Update the Settings Page UI:

2. Add new UI elements (like switches, dropdowns, or text fields) for the new preference on the Settings page. Use appropriate widgets to collect user input.
Ensure the layout is modular and follows design guidelines for consistency and usability.

3. Handle User Input:
Implement listeners for the new UI elements to store the updated preferences in the app's SharedPreferences or DataStore.
Ensure that changes take effect immediately or on the next startup, as needed.

**Comment by AdityaKBhadragond14** [comment]
Hi @icrc-fdeniger,

By notifications do you mean the toast messages that appear at the bottom or are there any in-app notifications?

**Comment by parthfloyd** [comment]
@icrc-fdeniger let me know if i shall close this issue as #101 PR is merged. I wanted to confirm regarding the documentation part mentioned in the description.

**Comment by icrc-fdeniger** [comment]
@parthfloyd  yes. I believe that the documentation is not mandatory here as the code seems straight forward

**Comment by parthfloyd** [comment]
Noted thanks!
